docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,268 @@
+---
+description: Learn how to send metric data to Grafana Mimir.
+keywords:
+  - send metrics
+  - native histogram
+  - prometheus
+  - grafana agent
+  - instrumentation
+menuTitle: Native histograms
+title: Send native histograms to Mimir
+weight: 100
+---
+
+# Send native histograms to Mimir
+
+Prometheus native histograms is a data type in the Prometheus ecosystem that enables producing, storing and querying high resolution [histogram](https://prometheus.io/docs/concepts/metric_types/#histogram) of observations.
+
+Native histograms are different from classic Prometheus histograms in a number of ways:
+
+1. Native histogram bucket boundaries are calculated with a formula depending on the scale (resolution) of the native histogram and are not user defined. The calculation produces exponentially increasing bucket boundaries. For more detail see [Bucket boundary calculation](#bucket-boundary-calculation).
+1. Native histogram bucket boundaries may change (widen) dynamically if observations result in too many buckets. For more detail see [Limiting the number of buckets](#limiting-the-number-of-buckets).
+1. Native histogram bucket counters only count observations inside the bucket boundaries, whereas the classic histogram buckets only had an upper bound called `le` and counted all observations in the bucket and all lower buckets (cumulative).
+1. An instance of a native histogram metric only requires a single time series, because the buckets, sum of observations, and count of observations are stored in a single new data type - and not in separate time series using the float data type. Thus there are no `<metric>_bucket`, `<metric>_sum`, `<metric>_count` series, only `<metric>`.
```

Comment:
```suggestion
* An instance of a native histogram metric only requires a single time series, because the buckets, sum of observations, and the count of observations are stored in a single data type rather than in separate time series using the `float` data type. Thus, there are no `<metric>_bucket`, `<metric>_sum`, and `<metric>_count` series. There is only `<metric>`.
```
I assume you do not mean `new` data type. :) What is the new data type called?

Reply:
right, I'll add native histogram which is the name of the new data type